Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -1,7 +1,4 @@
 category: Data Enrichment & Threat Intelligence
```

Comment:
Not clear to me why removal of arguments charecatsitics like default description isArray and such all across the file.
Please review it and revert if these changes are not required.